service/role.go

```diff
@@ -382,6 +372,27 @@ func (r *roleService) getRoleTokenAthenzURL(domain, role string, minExpiry, maxE
 	return u
 }
 
+func encode(domain, role, principal string) string {
+	roles := strings.Split(role, ",")
+	for i := range roles {
+		roles[i] = strings.TrimSpace(roles[i])
```

Comment:
no needs to trim?
https://github.com/yahoo/athenz/blob/12aefb788a92d4de44f8b4b78cefff648b16ae1c/core/zts/src/main/rdl/Names.tdl#L22
space is not allowed in this param

Reply:
fixed